include/targets/cuda/mma_tensor_op/smma_m16n8k8_sm70.cuh

```diff
@@ -0,0 +1,150 @@
+#pragma once
+
+#include <type_traits>
+#include <quda_fp16.cuh>
+#include <array.h>
+#include <trove/ptr.h>
+#include <mma_tensor_op/hmma_m16n8k8_sm70.cuh>
+
+namespace quda
+{
+  namespace smma
+  {
+
+    template <class shuffle_t, int inst_k_, int warp_m_, int warp_n_> struct smma_x_t {
+    };
+
+    template <> struct smma_x_t<mma::half, 8, 1, 1> {
+
+      static constexpr bool use_intermediate_accumulator() { return true; };
+
+      static __device__ __host__ constexpr int inline pad_size(int) { return 0; }
+
+      static constexpr bool do_rescale()
+      {
+        return true; // false because we use FP16
```

Comment:
typo on the comment

Reply:
https://github.com/lattice/quda/commit/d0da82373faf185f7b11fee05b7415cdbecf8424